lib/mapper.js

```diff
@@ -69,14 +72,23 @@ exports.screen = function(screen, settings){
  * @return {Object}
  */
 
+
 exports.track = function(track, settings){
   var result = createCommonGAForm(track, settings);
+
+  // page url
+  var url = track.proxy('properties.url') || track.proxy('context.page.url');
+  var parsed = parse(url || '');
+  if (parsed.hostname) result.dh = parsed.hostname;
+  if (parsed.path) result.dp = parsed.path;
```

Comment:
This logic is different than the logic in `#page`—some stuff captured there (e.g. referrer) is not captured here. Is there a reason for that? If not we should break this logic out into a helper similar to `createCommonGAForm`